Makefile.am

```diff
@@ -184,6 +184,13 @@ LIBNETDATA_FILES = \
     libnetdata/string/utf8.h \
     $(NULL)
 
+if ENABLE_DBENGINE
+    LIBNETDATA_FILES += \
+    libnetdata/global_uuid_map/global_uuid_map.c \
+    libnetdata/global_uuid_map/global_uuid_map.h \
```

Comment:
I believe those functions shouldn't be part of `/libnetdata`, cause no plugin that links against this code will ever use it. This code is exclusive to the core of the agent. Those functions are only called when dbengine code needs to be called as well.

One suggestion is under `database/engine/global_uuid_map`.

Reply:
Good point-- I will move it there, since it makes sense only with DBENGINE active anyway

Reply:
Moved